### DISCUSSION TITLE
How should winssh-pageant handle situations when a client like KeePassXC tries to add keys to it via the pageant protocol?

### DISCUSSION THREAD

**Discussion by ducksys** [ndbeals/winssh-pageant#20] [Ideas]
Hi @ndbeals 

Please take a look at this this issue https://github.com/keepassxreboot/keepassxc/issues/7919 starting from my first comment. What do you think?

**Comment by ndbeals** [comment]
Hey @ducksys, `winssh-pageant` _should_ support this. All it does is listen for a message from one place, and then relay that identical message to a different place. It _shouldn't_ matter what the message is, as long as both the sender and receiver understand it, this application should be transparent between them.
